### PR TITLE
Added icon for Stylelint

### PR DESCRIPTION
**Pull request by r3nya**
Hey!

I want to add icon for [Stylelint](http://stylelint.io/).
Need to handle `.stylelintrc` file.
Please look my changes.

### CODE REVIEW

File: src/icons.json [hunk 1]
```diff
@@ -829,6 +829,19 @@
       }
     ]
   },
+  "file_type_stylint": {
+    "color": "graphite",
+    "syntaxes": [
```
Comment (oivva) [inline src/icons.json]: This is an alias, not the syntax.

File: src/assets/file_type_stylelint.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...9" xmlns="http://www.w3.org/2000/svg" data-reactid="8"><g data-reactid="9"><path d="M20.272 38.16c4.375-15.642 4.248-15.713 7.41-26.866l.76.44c.414.237.752.042.752-.434V5.98c.84-2.938 1.496-5.21 1.69-5.89h3.425L40 5.36l-3.738 3.23 2.827 3.057L20.46 38.16c-.09.15-.217.116-.188 0z" data-reactid="10"/><path d="M22.307 4.444c.17 0 .306.093.306.21V7.38c0 .113-.137.206-.306.206h-4.6c-.17 0-.307-.093-.307-.208V4.652c0-.115.137-.21.306-.21h4.6zm5.68-3.018l-5.062 2.922c.058.09.094.193.094.305V7.38c0 .14-.057.268-.147.37l5.113 2.952c.348.202.634.037.634-.365V1.792c0-.403-.284-.567-.633-.366zM16.993 7.38V4.652c0-.112.036-.215.094-.304l-5.062-2.924c-.348-.2-.634-.036-.634.366v8.545c0 .402.287.567.636.365l5.112-2.952c-.09-.103-.144-.23-.144-.37z" data-reactid="11"/><path d="M19.54 38.16h.188c-4.374-15.642-4.248-15.713-7.41-26.866l-.76.44c-.413.237-.752.042-.752-.434V5.98C9.966 3.043 9.31.77 9.116.09H5.692L0 5.36l3.737 3.23L.91 11.648 19.54 38.16z" data-reactid="12"/><ellipse cx="19.861" cy="11.897" rx="1.382" ry="1.382" data-reactid="13"/><ellipse cx="19.861" cy="20.112" rx="1.382" ry="1.382" data-reactid="14"/><ellipse cx="19.861" cy="28.329" rx="1.382" ry="1.382" data-reactid="15"/></g></svg>
```
Comment (oivva) [inline src/assets/file_type_stylelint.svg]: Please read [icon requirements](https://github.com/oivva/st-file-icons/blob/dev/.github/CONTRIBUTING.md#icon). You have to edit your svg icon.

**Comment by r3nya** [comment]
So … I updated svg icon and rules. Please review it.

**Comment by oivva** [comment]
@r3nya Looks good, just change name `Stylelint` -> `JSON (Stylelint)`. This is an alias, so we need specify base syntax in the name.

```js
  "file_type_stylelint": {
    "color": "graphite",
    "aliases": [
      {
        "name": "JSON (Stylelint)",
        "extensions": [
          ".stylelintrc"
        ],
        "base": "source.json",
        "scope": "source.json.stylelint"
      }
    ]
  },
```

**Comment by oivva** [comment]
Thanks! :+1:

**Comment by r3nya** [comment]
@oivva thank u too. 😉

**Pull request merged**